ui/package.json

```diff
@@ -9,42 +9,42 @@
     "test": "jest --colors --verbose --passWithNoTests"
   },
   "dependencies": {
-    "axios": "^0.19.2",
-    "clipboard-copy": "^3.1.0",
+    "axios": "^0.21.1",
+    "clipboard-copy": "^4.0.1",
     "vue": "^2.6.12",
-    "vue-class-component": "^7.2.3",
-    "vue-monaco": "^1.2.1",
-    "vue-property-decorator": "^8.4.2",
-    "vue-router": "^3.3.2",
-    "vuetify": "^2.2.33",
-    "vuex": "^3.4.0",
+    "vue-class-component": "^7.2.6",
+    "vue-monaco": "^1.2.2",
+    "vue-property-decorator": "^9.1.2",
+    "vue-router": "^3.5.1",
+    "vuetify": "^2.4.5",
+    "vuex": "^3.6.2",
     "vuex-class": "^0.3.2"
   },
   "devDependencies": {
-    "@mdi/font": "^4.9.95",
-    "@types/jest": "^24.9.1",
-    "@types/node": "^12.12.47",
-    "@typescript-eslint/eslint-plugin": "^4.14.1",
-    "@typescript-eslint/parser": "^4.14.1",
-    "@vue/cli-plugin-babel": "^4.5.7",
-    "@vue/cli-plugin-typescript": "^4.5.7",
-    "@vue/cli-plugin-unit-jest": "^4.5.7",
-    "@vue/cli-service": "^4.5.7",
+    "@mdi/font": "^5.9.55",
+    "@types/jest": "^26.0.20",
+    "@types/node": "^14.14.31",
+    "@typescript-eslint/eslint-plugin": "^4.15.2",
+    "@typescript-eslint/parser": "^4.15.2",
+    "@vue/cli-plugin-babel": "^4.5.11",
+    "@vue/cli-plugin-typescript": "^4.5.11",
+    "@vue/cli-plugin-unit-jest": "^4.5.11",
+    "@vue/cli-service": "^4.5.11",
     "@vue/eslint-config-typescript": "^7.0.0",
-    "@vue/test-utils": "^1.1.0",
+    "@vue/test-utils": "^1.1.3",
     "babel-core": "7.0.0-bridge.0",
-    "eslint": "^7.18.0",
+    "eslint": "^7.20.0",
     "eslint-config-standard-with-typescript": "20.0.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-vue": "^7.0.1",
-    "monaco-editor-webpack-plugin": "^1.9.0",
-    "sass": "^1.26.8",
-    "ts-jest": "^24.3.0",
-    "typescript": "^3.9.7",
+    "eslint-plugin-promise": "^4.3.1",
+    "eslint-plugin-vue": "^7.6.0",
+    "monaco-editor-webpack-plugin": "^3.0.0",
```

Comment:
We can not update `monaco-editor-webpack-plugin` to `3.0.0` because it only works with `monaco-editor` `0.22.x`. We are using `vue-monaco` which currently is using `monaco-editor` `0.20.0`. Therefore we must stick with version `1.9.0`. Also see the [version compatibility matrix](https://github.com/microsoft/monaco-editor-webpack-plugin#version-matrix)  for `monaco-editor-webpack-plugin`.

Reply:
I think I found a way how it should work in commit 2988db7

Reply:
Not sure though, if it is a good idea to use this workaround..

Reply:
Well, even if the build now works, I do not think it is a good idea to fix this that way.

With your change we will have both version of `monaco-editor` in our dependency tree and `vue-monaco` will still use the old version `0.20.0`...

Reply:
I also think it is not very nice, because with your change we will have both versions of `monaco-editor` in the dependency tree. The older version is referenced by `vue-monaco` and the newer by `monaco-editor-webpack-plugin`. 

I just had a look into `vue-monaco` and I think your workaround will indeed work because `monaco-editor-webpack-plugin` will replace the old `monaco-editor` version from `vue-monaco` during build time by the new version.

Your change will also result in a longer installation and build times, because two `monaco-editor` versions must be downloaded and `monaco-editor` is a very large package.

---

`vue-monaco` is actually just a tiny wrapper around the `monaco-editor` with only 165 loc. To solve this issue we could also just copy the files from `vue-monaco` and remove the dependency. `vue-monaco` is MIT licenced so this is not an issue, is it?

Reply:
Hm, I agree that the solution is not optimal... I'd not copy it because we would not profit from new versions.. Well, I think we don't require the version upgrade by all means, so I'll downgrade `monaco-editor-webpack-plugin` back to `1.9.0`